src/App.tsx

```diff
@@ -39,8 +40,9 @@ class App extends Component<AppProps, {}> {
   render() {
     return (
       <div className={styles.App}>
-        <Route exact path="/" component={GridLayout} />
-        <Route exact path="/full/:id" component={FullLayout} />
+        <Route exact path="/" component={LandingPage} />
+        <Route exact path="/:dashboardName" component={GridLayout} />
+        <Route exact path="/:dashboardName/full/:id" component={FullLayout} />
       </div>
```

Comment:
Here I suspect there will be a problem on staging and production if the urls will have "dashboards" in them like this:

nxt3.staging.lizard.net/dashboards/ijgenzon1

Reply:
I think it should work but I cannot tell for sure. The base URL for the tile dashboard will be configured to be "nxt3.staging.lizard.net/dashboards/" so I expect it to work without having "dashboards" in the paths. Anyway, apart from deploying it to staging and configuring the backend to test if it works, do you know any other way that we can test it?

Reply:
I do currently not know a better way to test then deploying to staging. 
The fact that it does not work on dev makes me suspect it will not work on staging.

Reply:
But on dev, we expect to have the "/" represents for "nxt3.staging.lizard.net/dashboards/" on staging. Just like in the original lizard-tile-dashboard, on dev, we have "/" represents for "nxt3.staging.lizard.net/dashboard/". But we have to test it to know whether it is gonna work or not anyway :)